src/Symfony/Component/DependencyInjection/ContainerBuilder.php

```diff
@@ -1548,6 +1573,8 @@ public static function getServiceConditionals($value): array
      * @param mixed $value An array of conditionals to return
      *
      * @internal
+     *
+     * @return string[]
```

Comment:
please put that before `@internal` (same above)

Reply:
Done.